src/libstd/num/f32.rs

```diff
@@ -652,6 +652,58 @@ impl f32 {
         unsafe { cmath::nextafterf(self, other) }
     }
 
+    /// Returns floating point exponent of the input
+    ///
+    /// ```
+    /// use std::f32;
+    ///
+    /// let x = 0.5f32;
+    /// let abs_diff = (x.logb() - 1).abs();
+    ///
+    /// assert!(abs_diff <= f32::EPSILON)
+    /// ```
+    #[unstable(feature = "std_misc",
```

Comment:
Could these functions also be introduced under a new feature name? We're trying to cut down on the number of members of the `std_misc` feature, and something like `float_extras` may work well here.